.github/ISSUE_TEMPLATE/1.bug.yml

```
@@ -34,17 +34,17 @@ body:
     attributes:
       label: Steps to reproduce 🕹
       description: |
-        Provide a link to a live example (you can use codesandbox.io) and an unambiguous set of steps to reproduce this bug.
-        Include code to reproduce, if relevant (which it most likely is).
+        Please provide a link to a live example and an unambiguous set of steps to reproduce this bug.
+             
+        As a starting point, we recommend you browse our [documentation](https://mui.com/x/advanced-components/), and [select](https://mui.com/static/docs/forking-an-example.png) the closest example to your use case.
```

This link is a 301

```suggestion
        As a starting point, we recommend you browse our [documentation](https://mui.com/x/introduction/), and [select](https://mui.com/static/docs/forking-an-example.png) the closest example to your use case.
```

---

"the documentation". We document ourselves [here](https://mui.com/about/) 😁

```suggestion
        As a starting point, we recommend you [browse the documentation](https://mui.com/x/introduction/), and [select](https://mui.com/static/docs/forking-an-example.png) the closest example to your use case.
```